profile/get.go

```diff
@@ -266,13 +192,13 @@ func prepProfile(r record.Record) (*Profile, error) {
 	// prepare config
 	err = profile.prepConfig()
 	if err != nil {
-		log.Warningf("profiles: profile %s has (partly) invalid configuration: %s", profile.ID, err)
+		log.Errorf("profiles: profile %s has (partly) invalid configuration: %s", profile.ID, err)
 	}
 
 	// parse config
 	err = profile.parseConfig()
 	if err != nil {
-		log.Warningf("profiles: profile %s has (partly) invalid configuration: %s", profile.ID, err)
+		log.Errorf("profiles: profile %s has (partly) invalid configuration: %s", profile.ID, err)
 	}
```

Comment:
If that is an error I guess we should `return nil, err`?

Reply:
Let's talk about that.